src/__Function_test__/__snapshots__/Calculators.test.js.snap

```diff
@@ -0,0 +1,2107 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`it  crashing 1`] = `
+Object {
+  "asFragment": [Function],
+  "baseElement": <body>
+    <div>
+      <div
```

Comment:
- [ ] There should be the usage of Jest snapshots for testing all the components. You can add `--coverage` to Identify which lines have not been covered. Kindly add more snapshot testing
![image](https://user-images.githubusercontent.com/81932478/201467042-bc0b88c6-360e-4e28-b2e7-c99ffacdbfe2.png)